data/dataset/dynamodb_example_test_dataset.yml

```diff
@@ -0,0 +1,62 @@
+dataset:
+  - fides_key: dynamodb_example_test_dataset
+    name: DynamoDB Example Test Dataset
+    description: Example of a DynamoDB dataset containing a single customer table
+    collections:
+      - name: customer_identifier
```

Comment:
Hey! It's that linking table we talked about

Reply:
I really wanted to try and that in as a test case like we discussed 🤗